sdk-actors/src/main/java/io/dapr/actors/runtime/AbstractActor.java

```diff
@@ -244,9 +244,9 @@ protected ActorStateManager getActorStateManager() {
    * Resets the cached state of this Actor.
    */
   void rollback() {
-    if (!this.started) {
-      throw new IllegalStateException("Cannot reset state before starting call.");
-    }
+    //    if (!this.started) {
+    //      throw new IllegalStateException("Cannot reset state before starting call.");
```

Comment:
Why is this needed?

Reply:
This was just a quick hack to get a PoC working so I could publish the PR and get early feedback. If left in the way it is though, it explicitly forbids reentrancy.